### DISCUSSION TITLE
Wiring pin help

### DISCUSSION THREAD

**Discussion by gurmukhp** [kellerza/sunsynk#227] [Q&A] [answered]
Hi all

This is such a useful resouce, I have the SunSynk ECCO being installed in a few weeks and am looking forward to running this integration in Home Assistant!

After looking at the [docs](https://kellerza.github.io/sunsynk/guide/wiring), it's not clear to me what colour wire should be attached  to the USB adaptor.

The CAT5 cable I have is T568A, so does that mean I only need to connect the Green-White wire to the B port on the WAVE USB device?

Thanks!

**Comment by gurmukhp** [comment, marked as answer]
If this helps anyone else:

Pin 1 (The left most wire) connects to B
Pin 2 (The next on the left) connects to A
No need for ground

**Reply by kellerza** [reply]
No need for ground, but you will get less timeouts with ground wired up
